guides/03_building-with-blocks/05_more-blocks-features

```diff
@@ -70,3 +70,9 @@ In the  2 player tic-tac-toe demo below, a user can select a cell in the `DataFr
 
 $code_tictactoe
 $demo_tictactoe
+
+## Multipage Apps
+
+Your Gradio app can support multiple pages with the `gr.Blocks().route()` method. 
```

Comment:
I would suggest creating a separate guide for multipage apps where we discuss multipages in more depth including the fact that you can't share components, event listeners, otherwise you should use tabs, etc. Its likely we'll expand on multipages as well with dynamic routes and so on so worth having its own section imo and also for seo purposes.